packages/panels/src/Commands/MakePageCommand.php

```diff
@@ -47,8 +48,35 @@ public function handle(): int
         $resourceClass = null;
         $resourcePage = null;
 
-        $resourceInput = $this->option('resource') ?? text(
-            label: 'What is the resource you would like to create this in?',
+        $panel = $this->option('panel');
+
+        if ($panel) {
+            $panel = Filament::getPanel($panel);
+        }
+
+        if (! $panel) {
+            $panels = Filament::getPanels();
+
+            /** @var Panel $panel */
+            $panel = (count($panels) > 1) ? $panels[select(
+                label: 'Which panel would you like to create this in?',
+                options: array_map(
+                    fn (Panel $panel): string => $panel->getId(),
+                    $panels,
+                ),
+                default: Filament::getDefaultPanel()->getId()
+            )] : Arr::first($panels);
+        }
+
+        $resourceInput = $this->option('resource') ?? suggest(
+            label: 'Which resource would you like to create this in?',
+            options: collect($panel->getResources())
+                ->filter(fn ($path) => str_contains($path, '\\Resources\\'))
```

Comment:
```suggestion
                ->filter(fn (string $path): bool => str($path)->contains('\\Resources\\'))
```

Reply:
Type hints definitely, but wouldn't the native `str_contains` suffice? Using `str` helper seems a bit overkill.

Reply:
I don't like the readability, the order of arguments is ambiguous